python/ray/train/v2/_internal/state/schema.py

```diff
@@ -33,6 +33,10 @@ class RunStatus(str, Enum):
     ABORTED = "ABORTED"
 
 
+def is_terminal_run_status(status: RunStatus) -> bool:
+    return status in [RunStatus.FINISHED, RunStatus.ERRORED, RunStatus.ABORTED]
```

Comment:
nit: could we add these helpers as methods on the enum instead?

Reply:
Done. Tried adding is_terminal to the constructor like https://github.com/ray-project/ray/blob/master/python/ray/train/v2/_internal/execution/controller/state.py#L35 but that messed up printing and figured this was overengineering.